inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/repositories/AccountRepository.kt

```diff
@@ -74,6 +78,14 @@ internal abstract class AccountRepository {
             return false
         }
 
+        @SuppressLint("BinaryOperationInTimber")
+        override fun logWarningForUserInfo(tag: String, timber: Timber.Tree) {
+            if (getRaeToken().isNotEmpty() && getIdTrackingIdentifier().isNotEmpty()) {
```

Comment:
Does it work for all apps of only `jp.co.rakuten...` apps?
Can we also add a warning for enforcing userId when accessToken is provided?

Reply:
What do you mean by all apps? There should be no package name restriction.

I'll add the additional condition.